bundled_scripts/ilo.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+import sys
+from copy import deepcopy
+
+import hpilo
+
+
+USER = os.environ['MANAGEMENT_USER_NAME']
+PASS = os.environ['MANAGEMENT_USER_PASSWORD']
+
+MAC_PREFIX_BLACKLIST = [
+    '505054', '33506F', '009876', '000000', '00000C', '204153', '149120',
+    '020054', 'FEFFFF', '1AF920', '020820', 'DEAD2C', 'FEAD4D',
+]
+
+DEVICE_INFO_TEMPLATE = {
+    "model_name": "",
+    "processors": [],
+    "mac_addresses": [],
+    "disks": [],          # unused (hpilo doesn't provide such info)
+    "serial_number": "",
+    "memory": [],
+}
+
+PROCESSOR_TEMPLATE = {
+    "model_name": "",  # unused (hpilo doesn't provide such info)
+    "family": "",      # hpilo returns int here, but only for iLO2 (for iLO3 this field is empty)  # noqa
+    "label": "",
+    "index": None,     # unused, but similar info is available as "label"
+    "speed": None,
+    "cores": None,
+}
+
+MEMORY_TEMPLATE = {
+    "label": "",
+    "size": None,
+    "speed": None,
+    "index": None,  # unused, but similar info is available as "label"
+}
+
+
+class IloError(Exception):
+    pass
+
+
+def normalize_mac_address(mac_address):
+    mac_address = mac_address.upper().replace('-', ':')
+    return mac_address
+
+
+def get_ilo_instance(host):
+    ilo = hpilo.Ilo(hostname=host, login=USER, password=PASS, port=443)
```

Comment:
Actually, I've just noticed that it is not needed here - `python-hpilo` uses this port as default one (see [the docs](http://pythonhosted.org/python-hpilo/python.html)). I'll remove it.